doc/chapter-firststeps.asciidoc

```diff
@@ -40,7 +40,7 @@ need to write it as _%40_ instead so that it can be distinguished from the _@_
 that separates the username/password part from the hostname part.
 
 You can also configure local files as feeds, by prefixing the local path with
-`file://` and adding it to the _$HOME/.newsboat/urls_ file:
+`file://` and adding it to the _urls_ file:
```

Comment:
How about we link to the "XDG Base Directory Support" chapter whenever we refer to one of the files? That's probably a bit more user-friendly if one wants to find out the location of the file.

Just thinking out loud: Maybe we could have this chapter even renamed to a more generic "Files" to make it more visible in the table of contents. However, then the chapter needs to be restructured to suit the title. So probably not worth the hassle.

Reply:
Linking every time seems excessive. I understand how it can help a person who reads a single paragraph to figure out just one little thing they need, but still.

I like your suggestion about renaming the chapter. Let's do that; I think it also alleviates some of the need for links, since the chapter will be easy to find even without a link.

Reply:
I disagree, in my opinion a good documentation is also heavily crosslinked, so one can navigate to the interesting parts quickly. This is especially useful the larger documents grow. Hyperlinks don't hurt people who don't want to follow them, but certainly help others. But I'm fine with the current solution.

Reply:
Okay, filed #910 about that.